.github/workflows/apk.yml

```diff
@@ -64,26 +68,27 @@ jobs:
       - name: Upload APK
         uses: actions/upload-artifact@v3
         with:
-          name: "${{ github.event.inputs.name }}-Debug-APK"
-          #path: app/build/outputs/apk/release/*.apk
-          path: app/build/outputs/apk/debug/*.apk
+          name: "${{ github.event.inputs.name }}-Release-APK"
+          path: app/build/outputs/apk/release/*.apk
+
       - name: Create GitHub release
         id: create_release
         uses: actions/create-release@v1
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN}}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          tag_name: ${{ github.run_number}}
-          release_name: ${{ github.event.inputs.name }} (#${{ github.run_number}})
+          tag_name: ${{ github.run_number }}
+          release_name: ${{ github.event.inputs.name }} (#${{ github.run_number }})
```

Comment:
Are the space really necessary or is it a mistake ?

Reply:
It looks nicer that way